visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.html

```diff
@@ -1,9 +1,11 @@
-<md-card id="header">
+<search-panel-component></search-panel-component>
+
+<md-card class="header" id="header">
```

Comment:
Think about class and id name *header*. @alexhunziker and me would rather rename it to metrics-panel. Can we possibly remove the id or is it used somewhere necessarily?

Reply:
it is used for selecting which part of the ribbon bar is expanded when clicking on the section titles/the dedicated expansion button. Furthermore it is used for the e2e tests. Therefore i renamed the id instead